src/components/List.jsx

```diff
@@ -43,14 +93,29 @@ export const List = ({ token }) => {
             <li key={doc.id}>
               <input
                 type="checkbox"
-                checked={calcTimeDiff(doc.data().purchased_date)}
-                disabled={calcTimeDiff(doc.data().purchased_date)}
+                checked={calcTimeDiff(doc.data().last_purchased_date)}
+                disabled={calcTimeDiff(doc.data().last_purchased_date)}
                 name={doc.id}
                 id={doc.id}
                 onClick={(e) => handleClick(doc, e)}
                 onChange={(e) => handleClick(doc, e)}
               />
               <label htmlFor={doc.id}>{doc.data().item}</label>
+              {doc.data().total_purchases > 0 ? (
+                <p> Total purchases: {doc.data().total_purchases}</p>
+              ) : null}
```

Comment:
Not a must change. For future ref, this is a common pattern to use when your conditional for false is null or do nothing.
```suggestion
              {doc.data().total_purchases > 0 && <p> Total purchases: {doc.data().total_purchases}</p>}
```
More info on [MDN Logical AND](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND)

Reply:
Thanks Yenly. Good call.

Reply:
I see, the short circuit &&. That took a moment. Thank you!

Reply:
Committed the changes